scripts/sysman/device.yml

```diff
@@ -100,14 +117,40 @@ members:
       desc: "[out] Indicates if the device has been repaired"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Device reset properties"
+class: $sDevice
+name: $s_reset_properties_t
+base: $s_base_properties_t
+members:
+    - type: ze_bool_t
+      name: force
+      desc: |
+            [in] If set to true, all applications that are currently using the device will be forcibly killed.
+    - type: $s_reset_type_t
+      name: reset_type
+      desc: "[in] Type of reset needs to be performed"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Device universal unique id (UUID)"
+version: "1.7"
+class: $sDevice
+name: $s_uuid_t
+base: $s_base_properties_t
```

Comment:
`$s_uuid_t` should not inherit from `$s_base_properties_t`

Reply:
Fixed